src/Microsoft.OData.Core/ODataErrorDetail.cs

```diff
@@ -22,5 +24,16 @@ public sealed class ODataErrorDetail
         /// <summary>Gets or sets the target of the particular error.</summary>
         /// <returns>For example, the name of the property in error</returns>
         public string Target { get; set; }
+
+        /// <summary>
+        /// Serilization to Json format string.
+        /// </summary>
+        /// <returns>The string in Json format</returns>
+        public override string ToString()

```

Comment:
>public override string ToString() [](start = 8, length = 33)

is this class only ever used in the Json case? if not, we shouldn't be assuming json in the tostring override.